app/migration_v_0_7.go

```diff
@@ -0,0 +1,24 @@
+package app
```

Comment:
FYI, our next package version will be 1.x.x since it's breaking from 0.6.x and we ended up triggering a 0.7.x release. This will also bring it in line with semantic versioning, so this handler will need to be renamed.

Reply:
Will fix it later in a separate PR.